app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1 class="text-center py-4">Recipes list</h1>
+
+<%= link_to "Add New Recipe", new_recipe_path, class: "btn btn-primary mb-2 float-end" %>
+<% @recipes.each do |recipe| %>
+  <div class="card border-secondary mb-2" style="width: 100%;">
+    <div class="card-body text-secondary d-flex justify-content-between align-items-center">
+      <div style="width: 30%;">
+      <h5 class="card-title"><a href="<%= recipe_path(recipe) %>" class="text-decoration-none text-uppercase font-weight-bold"><%= recipe.name %></a></h5>
+      <%= button_to "REMOVE", recipe_path(recipe), method: :delete, data: { turbo_confirm: "Are you sure? "}, class: "text-decoration-none btn btn-danger" %>
+    </div>
+    <p style="width: 70%;"><%= recipe.description %></p>
+  </div>
+</div>
+<% end %>
```

Comment:
![image](https://github.com/GhausAliKhan/recipe-app/assets/93039359/6f1af7e6-54a6-43c5-9096-63fbdc965e09)

- [ ] It would be nice if you fix the total amount of the recipe here because I have an ingredient sugar of 10 dollars per unit and I have 20 units of sugar in this recipe so the total amount of the recipe should be 200 not 10 :thinking: Kindly fix this issue to make your app work as expected :+1: .